spec/sneakers/worker_handlers_spec.rb

```diff
@@ -103,6 +103,31 @@ def process(*args,&block)
 
   describe 'Maxretry' do
     let(:max_retries) { nil }
+    let(:props_with_x_death_count) {
+      {
+        :headers => {
+          "x-death" => [
+                        {
+                          "count" => 3,
+                          "reason" => "expired",
+                          "queue" => "downloads-retry",
+                          "time" => Time.now,
+                          "exchange" => "RawMail-retry",
+                          "routing-keys" => ["RawMail"]
+                        },
+                        {
+                          "count" => 3,
```

Comment:
Do these two counts need to be the same? It might be more illustrative to have them be separate so your test below is verifying that you're only using the count associated with the queue you're interested in.